.github/workflows/test-npm-run.yml

```diff
@@ -19,12 +19,12 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        node-version: [16.x]
+        node-version: [16.x, 20.x]
```

Comment:
@quesabe why not keeping just 20?

Reply:
I was not confident the version 16 is not used anywhere downstream. Hence I left the check to ensure the actions still work with node@16.

If you believe we can drop any support for old versions, I can remove v16 and in the future save some resources on these tests.